traffic_ops/client/servercheck.go

```diff
@@ -0,0 +1,36 @@
+/*
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package client
+
+import (
+	"encoding/json"
+
+	"github.com/apache/trafficcontrol/lib/go-tc"
+)
+
+const API_V13_SERVERCHECK = "/api/1.3/servercheck"
+
+// Update a Server Check Status
+func (to *Session) UpdateCheckStatus(status tc.ServercheckNullable) (*tc.ServercheckPostResponse, error) {
```

Comment:
Mind changing this to return `ReqInf`? 

Should just be 

`func (to *Session) UpdateCheckStatus(status tc.ServercheckNullable) (*tc.ServercheckPostResponse, ReqInf, error) {`

```
reqInf, err = post(to, uri, jsonReq, &resp)
if err != nil {
		return nil, reqInf, err
	}
	return &resp, reqInf, nil
```

We're working to change all client funcs to return `ReqInf`. It has data that helps clients debug and return better error messages.

Reply:
Sure, will do.